code/game/machinery/doors/airlock.dm

```diff
@@ -870,29 +870,69 @@ About the new airlock wires panel:
 
 	qdel(src)
 
+/obj/machinery/door/airlock/proc/Hulk_Jump(mob/living/carbon/user)

```

Comment:
А ещё вот что успел нагуглить по твоему запросу
>For instance one of the major functions called by SendMaps()--which is itself a fairly large function--gets inlined because it's called on only the one place.